packages/esm-patient-chart-app/src/index.ts

```diff
@@ -112,6 +113,11 @@ export const stopVisitPatientSearchActionButton = getSyncLifecycle(stopVisitActi
   moduleName,
 });
 
+export const patientProgramSummary = getSyncLifecycle(PatientSummaryOverviewList, {
```

Comment:
The data is related to a program but some people may classify it differently.

Reply:
It is not program specific, I think the name might be misleading , it’s more of a clinical view summary. I have updated the name

Reply:
Part of my point is that this is a public interface intended to be consumed by multiple configurations written by implementations that may or may not understand what the original purpose of a widget is; thus naming things so that they are consistent, discoverable, and not confusing is of vital importance here.